packages/vinxi/lib/build.js

```diff
@@ -155,6 +155,7 @@ export async function createBuild(app, buildConfig) {
 							dir: router.dir,
 							baseURL: router.base,
 							passthrough: true,
+							fallthrough: true,
```

Comment:
Not sure about the right default for all the router modes.

It seems nitro sets fallthrough to false under some conditions. For spa's this should be true otherwise 404 is returned for non existing paths instead of index.html.

Should this option be exposed on the router config? 
Should the public asset entry be generated for every spa router by default, removing the need to define it explicitly?

```js
{
  mode: "spa",
  publicDir: false
},
{
  mode: "spa",
  publicDir: {
    fallthrough: false
  }
}
```

Reply:
passthrough and fallthrough are different things ?? that's so confusing.

Can you explain which one does what? or is passthrough nothing lol.

Reply:
You are right I think `passthrough` isn't a thing... 🤣 can't find it in nitro or h3.
Ok so just have to replace it with `fallthrough` instead.